mne_bids/tests/test_mne_bids.py

```diff
@@ -68,6 +70,21 @@ def test_fif():
     raw_to_bids(subject_id=subject_id2, run=run, task=task, acquisition=acq,
                 session_id=session_id, raw_file=raw_fname2,
                 events_data=events_fname, output_path=output_path,
+                event_id=event_id, overwrite=False)
+    # check that the overwrite parameters work correctly for the participant
+    # data
+    # change the gender but don't force overwrite.
+    raw.info['subject_info'] = {'his_id': subject_id2,
+                                'birthday': (1994, 1, 26), 'sex': 2}
+    with pytest.raises(OSError, match="already exists"):
```

Comment:
@monkeyman192 actually why not raise an `IOError` here? It seems that will be more consistent with what [mne does](https://github.com/mne-tools/mne-python/blob/935866b466da206569cc508c65c003de9a36c65c/mne/utils.py#L2280)

Reply:
you mean all the `OSError`'s? I had them because the actual error I wanted doesn't exist in python 2.7, (`FileExistsError`), so I used what it is a pseudonym for which is `OSError(errno.EEXIST)`

Reply:
on this topic, [this](https://www.python.org/dev/peps/pep-3151/) is probably good/interesting reading...

Reply:
okay I'm not sure what is the recommendation by this PEP. It does mention that `IOError` is far more common ...

anyway, I am fine with this if you insist :)